### PR TITLE
update readme with user docs

### LINKED ISSUE #60: update README with content from core repo
Related to https://github.com/openwhisk/openwhisk/issues/1848

### PR DESCRIPTION
**Pull request by csantanapr**
Fixes #60

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,29 +1,221 @@
-## Status
-This is still very much a work-in-progress.
+## Using the Cloudant package
+The `/whisk.system/cloudant` package enables you to work with a Cloudant database. It includes the following actions and feeds.
 
-RESTful service to listen for changes to a Cloudant database.
+| Entity | Type | Parameters | Description |
+| --- | --- | --- | --- |
+| `/whisk.system/cloudant` | package | BluemixServiceName, host, username, password, dbname, overwrite | Work with a Cloudant database |
+| `/whisk.system/cloudant/read` | action | dbname, includeDoc, id | Read a document from a database |
```
Comment (jasonpet) [inline README.md]: should be /whisk.system/cloudant read-document.  includeDoc is not a parameter

File: README.md [hunk 2]
```diff
@@ -1,29 +1,221 @@
...
+| `/whisk.system/cloudant/write` | action | dbname, overwrite, doc | Write a document to a database |
```
Comment (jasonpet) [inline README.md]: we do not have a write action.  we have a create-document

File: README.md [hunk 3]
```diff
@@ -1,29 +1,221 @@
...
+          }
+      ]
+  }
+  ```
+
+### Writing to a Cloudant database
+
+You can use an action to store a document in a Cloudant database called `testdb`. Make sure that this database exists in your Cloudant account.
+
+1. Store a document by using the `write` action in the package binding you created previously. Be sure to replace `/myNamespace/myCloudant` with your package name.
```
Comment (jasonpet) [inline README.md]: no write action.

File: README.md [hunk 4]
```diff
@@ -1,29 +1,221 @@
...
+
+
+### Reading from a Cloudant database
+
+You can use an action to fetch a document from a Cloudant database called `testdb`. Make sure that this database exists in your Cloudant account.
+
+1. Fetch a document by using the `read` action in the package binding that you created previously. Be sure to replace `/myNamespace/myCloudant` with your package name.
+
+  ```
+  $ wsk action invoke /myNamespace/myCloudant/read --blocking --result --param dbname testdb --param id heisenberg
```
Comment (jasonpet) [inline README.md]: should be read-document

File: README.md [hunk 5]
```diff
@@ -1,29 +1,221 @@
...
+  return { "isWalter:" : doc.name === "Walter White"};
+}
+```
+
+Create the action to process the document from Cloudant:
+```
+$ wsk action create myAction myAction.js
+```
+
+To read a document from the database, you can use the `read` action from the Cloudant package.
```
Comment (jasonpet) [inline README.md]: should be read-document

File: README.md [hunk 6]
```diff
@@ -1,29 +1,221 @@
...
+The `read` action may be composed with `myAction` to create an action sequence.
```
Comment (jasonpet) [inline README.md]: same as above comment

File: README.md [hunk 7]
```diff
@@ -1,29 +1,221 @@
...
+```
+$ wsk action create sequenceAction --sequence /myNamespace/myCloudant/read,myAction
```
Comment (jasonpet) [inline README.md]: read-document

File: README.md [hunk 8]
```diff
@@ -1,29 +1,221 @@
...
+2. Verify that the document exists by browsing for it in your Cloudant dashboard.
+
+  The dashboard URL for the `testdb` database looks something like the following: `https://MYCLOUDANTACCOUNT.cloudant.com/dashboard.html#database/testdb/_all_docs?limit=100`.
+
+
+### Reading from a Cloudant database
+
+You can use an action to fetch a document from a Cloudant database called `testdb`. Make sure that this database exists in your Cloudant account.
+
+1. Fetch a document by using the `read` action in the package binding that you created previously. Be sure to replace `/myNamespace/myCloudant` with your package name.
```
Comment (jasonpet) [inline README.md]: read-document

**Comment by csantanapr** [comment]
@jasonpet it looks like we going to keep the `read` and `write` around until we can deprecate/remove them.
See changes to 41075d58a867a5bf0473048431379169ae5143c9

**Pull request merged**